css/style.css

```
@@ -0,0 +1,315 @@
+.visually-hidden {
+    position: absolute !important;
+    clip: rect(1px 1px 1px 1px);
+    clip: rect(1px, 1px, 1px, 1px);
+    padding: 0 !important;
+    border: 0 !important;
+    height: 1px !important;
+    width: 1px !important;
+    overflow: hidden;
+    display: none;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    font-family: Roboto;
+    font-size: 16px;
+    line-height: 24px;
+    color: #283136;
+}
+
+a {
+    text-decoration: none;
+}
+
+img {
+    max-width: 100;
+    height: auto;
+}
+
+ul {
+    list-style: none;
+}
+
+
+/* HEADER PART*/
+
+.slogan {
+    font-family: Roboto;
```

шрифт лучше вынести отдельно в боди. Он везде один. Иначе куча лишнего кода
```suggestion
   body{
    font-family: Roboto;
    }
```